doc/classes/Particles2D.xml

```diff
@@ -286,16 +286,16 @@
 	</methods>
 	<members>
 		<member name="amount" type="int" setter="set_amount" getter="get_amount">
-			Number of particles to emit.
+			Number of particles that can exist simultaneously and that are emitted in one emission cycle.
 		</member>
 		<member name="draw_order" type="int" setter="set_draw_order" getter="get_draw_order" enum="Particles2D.DrawOrder">
 			Particle draw order. Uses [code]DRAW_ORDER_*[/code] values. Default value: [code]DRAW_ORDER_INDEX[/code].
 		</member>
 		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
-			If [code]true[/code] particles are being emitted. Default value: [code]true[/code].
+			Emits particles if [code]true[/code]. Default value: [code]true[/code].
 		</member>
 		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio">
-			Time ratio between each emission. If [code]0[/code] particles are emitted continuously. If [code]1[/code] all particles are emitted simultaneously. Default value: [code]0[/code].
+			How rapidly particles in an emission cycle are emitted. If [code]0[/code] particles are emitted continuously and you wouldn't notice a cycle. If [code]0.5[/code] the whole [code]amount[/code] is emitted in half the time and there is a gap before the next cycle. If [code]1[/code] all particles are emitted immediately. Default value: [code]0[/code].
```

Comment:
"and you wouldn't notice a cycle" is redundant.
Changing "simultaneously" to "immediately" makes the description less accurate. "Immediately" implies a relative time, but relative to what? If you don't like the first wording, how about "all particles are emitted simultaneously at the beginning of the cycle"